src/cfnlint/customRules/Operators.py

```diff
@@ -9,28 +9,65 @@
 def equalsOp(template, rule, propertyList):
     """ Process EQUALS operators """
     matches = []
+    for prop in propertyList:
+        actualValue = getProperty(prop, rule)
+        if actualValue.strip().lower() != str(rule.value).strip().lower():
```

Comment:
Nice! New operators should be easier to implement now that they're sharing most of the code

---

Don't need to decide yet, but starting a thread here for thoughts on case sensitivity of these rules so we don't forget

Reply:
At the current moment, operators are case sensitive but values are not case sensitive. I'm open to either option though, or even making that configurable!

Reply:
Case sensitivity is a good point, it matters in some cases but not others. Maybe having two distinct operators, EQUALS and STRICT EQUALS?

Reply:
That sounds like a great idea - The rule class ingests all custom rule input first. That class could easily support both these cases, so that we do not have to clutter the main "Operator" dict.

Reply:
I think the case sensitivity decision affects most of the operators so far (`EQUALS`, `NOT_EQUALS`, `IN`, `NOT_IN`)

I agree that someone will eventually want the ability to choose either case sensitive or case insensitive enforcement

Regardless of the final decision, it should be consistent across all the applicable operators

Reply:
I'm looking at implementation of a "STIRCT_" in front of ANY operator where we want case sensitivity to be enforced. Common across all operators.

This can be done in the rule class to keep the operator class as neat and friendly as possible so anyone can add in an operator with minimal effort and knowledge!